src/main/kotlin/app/manager/exceptions/NotFoundException.kt

```diff
@@ -0,0 +1,3 @@
+package app.manager.exceptions
+
```

Comment:
We could add a response status for that exception with an annotation. Something like this:
```suggestion

@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "Snippet not found")
```